src/x86/runtime.rs

```diff
@@ -0,0 +1,120 @@
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[macro_export]
+macro_rules! __unstable_detect_feature_x86 {
+    ("sse") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse{}) };
+    ("sse2") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse2{}) };
+    ("sse3") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse3{}) };
+    ("ssse3") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::ssse3{}) };
+    ("sse4.1") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse4_1{}) };
+    ("sse4.2") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse4_2{}) };
+    ("avx") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::avx{}) };
+    ("avx2") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::avx2{}) };
+    ("fma") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::fma{}) };
+    ("bmi") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::bmi{}) };
+    ("bmi2") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::bmi2{}) };
+    ("abm") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::abm{}) };
+    ("lzcnt") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::abm{}) };
+    ("tbm") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::tbm{}) };
+    ("popcnt") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::popcnt{}) };
+}
+
+/// X86 CPU Feature
```

Comment:
Could this be `#[doc(hidden)]` with explicit documentation "please do not use this, it will break"?

Reply:
I've added it here, and also below to the other pub `__unstable_...` fn.